api/Window.json

```diff
@@ -2156,23 +2149,11 @@
                 ]
               }
             ],
-            "firefox_android": [
-              {
-                "version_added": "4",
-                "partial_implementation": true,
-                "notes": "By default, this method exists and can be called, but does nothing unless enabled in the browser's preferences."
-              },
-              {
-                "version_added": "4",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "browser.dom.window.dump.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              }
-            ],
+            "firefox_android": {
+              "version_added": "4",
+              "partial_implementation": true,
```

Comment:
I just reviewed and merged this change. Can you rebase and update the description to match what remains?